doc/src/examples/geometries/point.cpp

```diff
@@ -19,11 +19,12 @@ int main()
 {
     bg::model::point<double, 2, bg::cs::cartesian> point1;
     bg::model::point<double, 3, bg::cs::cartesian> point2(1.0, 2.0, 3.0); /*< Construct, assigning three coordinates >*/
-    point1.set<0>(1.0); /*< Set a coordinate. [*Note]: prefer using `bg::set<0>(point1, 1.0);` >*/
-    point1.set<1>(2.0);
 
-    double x = point1.get<0>(); /*< Get a coordinate. [*Note]: prefer using `x = bg::get<0>(point1);` >*/
-    double y = point1.get<1>();
+    bg::set<0>(point1, 1.0); /*< Set a coordinate. >*/
+    bg::set<1>(point1, 2.0);
+
+    double x = bg::get<0>(point1); /*< Get a coordinate. >*/
+    double y = bg::get<1>(point1);
 
```

Comment:
This sample explains the class point.cpp, not the point concept. Therefore it was written as it was. The note is I think clear enough: prefer use bg::set. But it is good to have an example of the methods of the class itself too.
Like point_xy which would give the usage of .x() and .y() (only there is no example there...)